src/main/java/me/zeroeightsix/kami/module/modules/misc/AntiAFK.kt

```diff
@@ -39,13 +39,15 @@ internal object AntiAFK : Module(
     private val walk = setting("Walk", true)
     private val radius by setting("Radius", 64, 8..128, 8)
     private val inputTimeout by setting("InputTimeout(m)", 0, 0..15, 1)
+    private val allowBreak by setting("AllowBreak", false)
 
     private var startPos: BlockPos? = null
     private var squareStep = 0
     private var baritoneDisconnectOnArrival = false
     private val inputTimer = TickTimer(TimeUnit.MINUTES)
     private val actionTimer = TickTimer(TimeUnit.TICKS)
     private var nextActionDelay = 0
+    private var wasAllowBreakAllowed = false
```

Comment:
this should follow the naming scheme of baritoneDisconnectOnArrival and move the var beside it as well